tools/appveyor.psm1

```diff
@@ -159,6 +159,10 @@ function Invoke-AppVeyorBuild
       }
       Start-PSBuild -FullCLR
       Start-PSBuild -CrossGen -Configuration $buildConfiguration
+      if(Test-DailyBuild)
+      {
+          Start-PSBuild -CrossGen -Configuration 'CodeCoverage'
+      }
```

Comment:
Building `-Configuration 'CodeCoverage'` will produce wrong packages with `Start-PSPackage` in the `finish` stage. This is because `Start-PSPackage` assumes the more recent build is targeting .net core and with `release` configuration, see [here](https://github.com/PowerShell/PowerShell/blob/master/build.psm1#L1126). You can move this if block before `Start-PSBuild -CrossGen -Configuration $buildConfiguration` or `Start-PSBuild -FullCLR` to work around it.

Moreover, [this line](https://github.com/PowerShell/PowerShell/blob/master/build.psm1#L1102) should be changed to `$Script:Options.Configuration -ne $Configuration -or` since `Configuration` takes new value now.